Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -24,9 +24,9 @@ $connection = Get-VssConnection -TaskContext $distributedTaskContext
 
 $resources = Get-EnvironmentResources -EnvironmentName $environmentName -ResourceFilter $machineNames -Connection $connection -ErrorAction Stop
 
-$machineUserName = Get-EnvironmentProperty -EnvironmentName $environmentName -Key "Username" -Connection $connection -ErrorAction Stop
+$machineUserName = Get-EnvironmentProperty -EnvironmentName $environmentName -Key "Microsoft-Vslabs-MG-Resource-Username" -Connection $connection -ErrorAction Stop
```

Comment:
This is still flaky and will break when machine level override is in place. Why don't you use the cmdlet and get environment object and then use env.resource.UserName and env.Resource.Password?